src/roll20/page-script.js

```diff
@@ -28,6 +28,23 @@ function updateHP(name, current, total, temp) {
     }
 }
 
+function updateCombatTracker(combat) {
+    const index = combat.findIndex(x => x[2]);
```

Comment:
Using object as suggested above would make this into `findIndex(combatant => combatant.turn)` which makes it more readable for future maintenance